bundles/pom.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Spotless complains about this file, so you should apply it here as well, I guess.

Reply:
strange, spotless was the one that made that change... I'll manually edit it back